tests/platform_tests/test_sequential_restart.py

```diff
@@ -34,13 +34,34 @@ def heal_testbed(duthosts, rand_one_dut_hostname):
         logging.info("Restoring dut with critical process failure: {}".format(details))
         config_reload(duthost, config_source='config_db', wait=120)
 
+def is_service_hiting_start_limit(duthost, container_name):
+    """
+    @summary: Determine whether the container can not be restarted is due to
+              start-limit-hit or not
+    """
+    service_status = duthost.shell("sudo systemctl status {}.service | grep 'Active'".format(container_name))
+    for line in service_status["stdout_lines"]:
+        if "start-limit-hit" in line:
+            return True
+
+    return False
+
 def restart_service_and_check(localhost, dut, service, interfaces):
     """
     Restart specified service and check platform status
     """
 
     logging.info("Restart the %s service" % service)
-    dut.command("sudo systemctl restart %s" % service)
+    for asic_index in dut.get_asic_ids():
```

Comment:
Prefer to use enum_asic_index instead here. You can enum the test to run against each individual asic.

Reply:
Updated to use the enum_frontend_asic_index and run test against each front end ASIC. 
I am not running with backend asic as that run will be similar result as front_end asic and we cannot check the interface/transciver status.